test/AutoDiff/stdlib/tgmath_derivatives.swift.gyb

```diff
@@ -38,17 +38,40 @@ func expectEqualWithTolerance<T>(_ expected: TestLiteralType, _ actual: T,
              file: file, line: line)
 }
 
+func computeDividedDifference<T: BinaryFloatingPoint> (
```

Comment:
I tried to use instead [symmetric derivative](https://en.wikipedia.org/wiki/Symmetric_derivative) formula to compute the numerical derivative, which gives a more precise result hence allowing to use lower values for `ulps` but some test cases for `reminder` broke, hence I dropped that change, but maybe we should use it in the future.